git-setup.sh

```diff
@@ -2,7 +2,7 @@
 # If no project name is given
 if [ $# -eq 0 ]; then
   # Display usage and stop
-  echo "Usage: git-setup.sh <console,website,platform>"
+  echo "Usage: git-setup.sh <website>"
```

Comment:
In `git-init.sh` we have this:

```
#!/bin/sh
# Skip if `.gitmodules` exists
[ -f .gitmodules ] && {
  echo ".gitmodules already initialized"
  exit 0
}

./git-setup.sh website console
```

Should we still keep `console` here?

Reply:
We could for now.